test/test_migrate.py

```diff
@@ -46,8 +47,9 @@ def check_migrate(orig, expected):
             f.write(orig.encode('utf-8'))
 
         run_qtile_migrate(config_path)
+        if have_mypy():
+            run_qtile_check(config_path)
```

Comment:
Doesn't seem to stop me doing my bad thing. Shouldn't we assert it?

Reply:
Yes, looks like we should. Also looks like the mypy env doesn't have bowler... bleh.

Reply:
Should we assert it? Wouldn't it be enough to run this check in one environment? I mean, we would be using the same bowler version in different test envs, that shouldn't make a difference and just makes the test runs slower...

Reply:
In theory environments shouldn't make a difference at all; in practice they often do :). Yes, we should put an assert here, though.